src/common/args.h

```diff
@@ -363,6 +364,12 @@ class ArgsManager
         m_network_only_args.clear();
     }
 
+    /**
+     * Check CLI command args
+     * (throws std::runtime_error when multiple CLI_COMMAND arguments are specified)
+     */
+    void CheckMultipleCLIArgs() const;
+
```

Comment:
nit:  If other changes are being made, could use `@throws` doxygen  syntax instead.

Currently renders as:
![image](https://github.com/user-attachments/assets/5fcb651f-d2d0-4ecd-b264-c5d287594ad1)

Using @throws for the exception:
![image](https://github.com/user-attachments/assets/2a3692fe-fb8a-43b8-a102-97c1b588b32d)


```diff
diff --git a/src/common/args.h b/src/common/args.h
index 73a4cca1441..f2489ea7446 100644
--- a/src/common/args.h
+++ b/src/common/args.h
@@ -366,7 +366,8 @@ protected:
 
     /**
      * Check CLI command args
-     * (throws std::runtime_error when multiple CLI_COMMAND arguments are specified)
+     * 
+     * @throws std::runtime_error when multiple CLI_COMMAND arguments are specified
      */
     void CheckMultipleCLIArgs() const;
```

Reply:
thanks, added https://github.com/bitcoin/bitcoin/pull/30148/commits/8838c4f171f3c3e568d237b216167fddb521d0a7